src/scenes/Main/Main.view.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import classNames from 'classnames';
+
+import { Title } from '@game/components';
+import * as Game from './components';
+
+import { IOptions } from '@game/types';
+
+import './style.scss';
+
+interface IMain {
+    options: IOptions;
+    winner: boolean;
+    turn: boolean;
+    field: string[];
```

Comment:
arrays should be fileds:?

Reply:
field will be removed due to the reason of directly using it in Fields.tsx file